nix/build-support/build_inputs.nix

```diff
@@ -0,0 +1,43 @@
+{
+  pkgs,
+  lib,
+  stdenv,
+  enableX11 ? stdenv.hostPlatform.isLinux,
+  enableWayland ? stdenv.hostPlaform.isLinux,
+}:
+[
+  pkgs.libGL
+]
+++ (lib.optionals stdenv.hostPlatform.isLinux [
```

Comment:
The parens are actually not required here

Reply:
Fixed.